playground/DurableTask/DurableTask.Scheduler.WebApi/Program.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Client.AzureManaged;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+
+builder.Services.AddDurableTaskClient(
+    clientBuilder =>
+    {
+        clientBuilder.UseDurableTaskScheduler(
+            builder.Configuration.GetConnectionString("taskhub") ?? throw new InvalidOperationException("Scheduler connection string not configured."),
+            options =>
+            {
+                options.AllowInsecureCredentials = true;
+            });
```

Comment:
Should we build a client integration?

Reply:
...yes?  It's complicated because consuming applications can be either DTS "clients" or "workers" (and sometimes both), and there are separate SDKs for each which means multiple client integrations.  Also, there are two sets of SDKs that apps can use (and you might consider another scenario a third) which further expands the matrix.

I'd say we should start with client integrations for the "modern" SDK, worker first and then client, as the former is would be the most common.  Then, if there's demand, look at integrations for the "older" SDKs.  That said, any client integrations would be follow up PRs.